scripts/imgtool/keys/ecdsa.py

```diff
@@ -102,37 +102,57 @@ def generate():
     def _get_public(self):
         return self.key.public_key()
 
-    def _build_minimal_ecdsa_privkey(self, der):
+    def _build_minimal_ecdsa_privkey(self, der, format):
+        if (format == serialization.PrivateFormat.OpenSSH):
```

Comment:
You don't need to use parentheses here.

Reply:
Done. I run PEP8 so I fixed all these issues related to parentheses. I also submitted a separate commit where I fixed the remaining PEP8 warnings in the files that I touched in this PR.